packages/mendel-manifest-uglify/manifest-uglify.js

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2015, Yahoo Inc.
+   Copyrights licensed under the MIT License.
+   See the accompanying LICENSE file for terms. */
+
+var UglifyJS = require("uglify-js");
+var debug = require("debug")('mendel-manifest-uglify');
+
+module.exports = manifestUglify;
+
+function manifestUglify(manifests, options, next) {
+    var optBundles = [].concat(options.bundles).filter(Boolean);
+    var uglifyOptions = Object.assign({
```

Comment:
this is a different default option than the uglify itself. Want to document this a little bit